_source/_posts/2023-06-08-identity-developer-podcast.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: blog_post
+title: "Unlock identity expertise with our new developer podcast!"
+author: edunham
+by: advocate
+communities: [devops]
+description: ""
+tags: []
```

Comment:
Recommend tags like "announcement, podcast, identity"

(podcast might be good because as you create podcast based posts, it will allow people to find them